package.json

```diff
@@ -12,13 +12,33 @@
   "bin": {
     "calypso-sdk": "./bin/sdk-cli.js"
   },
-  "browserslist": [
-    "last 2 versions",
-    "Safari >= 10",
-    "iOS >= 10",
-    "not ie <= 10",
-    "> 1%"
-  ],
+  "browserslist": {
+    "defaults": [
+      "last 2 versions",
+      "Safari >= 10",
+      "iOS >= 10",
+      "not ie <= 10",
+      "> 1%"
+    ],
+    "evergreen": [
```

Comment:
Would `last 2 versions, unreleased versions, not dead` rules cover these all at once?

https://github.com/browserslist/browserslist#full-list

Reply:
Unfortunately, that includes a bunch more browsers that are definitely not evergreen, such as IE11: https://browsersl.ist/?q=last+2+versions%2C+unreleased+versions%2C+not+dead